fawltydeps/settings.py

```diff
@@ -98,6 +98,47 @@ def parse_path_or_stdin(arg: str) -> PathOrSpecial:
     return Path(arg)
 
 
+DEFAULT_IGNORE_UNUSED = {
+    # Development tools not meant to be imported
+    # Formatting Tools
+    "autopep8",
+    "black",
+    "isort",
+    "pyformat",
+    "yapf",
+    # Linting Tools
+    "bandit",
+    "flake8",
+    "mccabe",
+    "pyflakes",
+    "pylint",
+    # Security Tools
+    "codespell",
+    # Documentation Tools
+    "sphinx",
+    "sphinx-rtd-theme",
+    "recommonmark",
+    # Testing Tools
+    "coverage",
+    "fawltydeps",
+    "hypothesis",
```

Comment:
I'm not sure about including `hypothesis`. Can it be used without being imported in the codE?

Reply:
I took a quick look at the introductory docs for Hypothesis, and AFAICS they always need `import hypothesis` somewhere in their examples.

Although there is a `hypothesis` command-line tool that is installed by the package, it seems to be geared towards writing/rewriting test code, and its output (e.g. from running `hypothesis write fawltydeps.utils`) is test code that does indeed `import hypothesis`.

So I think it makes sense to _remove_ `hypothesis` from our `DEFAULT_IGNORE_UNUSED` list.

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/939043f6fe59516cda60b0d13227d77f50fb114b.